spec/tests/helpers/perf_results_helper_spec.rb

```diff
@@ -508,5 +510,98 @@
       end
     end
   end
+
+  describe "#query_bigquery" do
+    let(:logger) { double }
+    let(:test_google_bigquery) { Class.new }
+    let(:test_bigquery_instance) { Class.new }
+    test_data = [{ "field" => "value" }]
+
+    before do
+      allow(subject).to receive(:logger).and_return(logger)
+      stub_const("Google::Cloud::Bigquery", test_google_bigquery)
+    end
+
+    context "when the GOOGLE_APPLICATION_CREDENTIALS environment variable is nil" do
+      it "raises an error" do
+        ENV["GOOGLE_APPLICATION_CREDENTIALS"] = nil
+        expect { subject.query_bigquery(TEST_BIGQUERY_SQL) }
+          .to raise_error(/GOOGLE_APPLICATION_CREDENTIALS/)
+      end
+    end
+
+    context "when the GOOGLE_APPLICATION_CREDENTIALS environment variable is empty" do
+      it "raises an error" do
+        ENV["GOOGLE_APPLICATION_CREDENTIALS"] = nil
```

Comment:
This test is the same as the one above...  I think you meant to replace nil with "" or something.

Reply:
Exactly; updated :)